var/spack/repos/builtin/packages/extrae/package.py

```diff
@@ -59,30 +65,35 @@ def install(self, spec, prefix):
         elif 'mvapich2' in spec:
             mpi = spec['mvapich2']
 
-        extra_config_args = []
+        config_args = ["--prefix=%s" % prefix,
```

Comment:
Would you mind converting this package to an `AutotoolsPackage`? Then you won't need `--prefix` and a lot of the `configure`, `make`, `make install` logic below.

Reply:
Done. Required conversion to an out of tree build and some other pain along the way thanks to autotools.